src/core/SpyStrategy.js

```diff
@@ -150,10 +150,13 @@ getJasmineRequireObj().SpyStrategy = function(j$) {
    * @name SpyStrategy#throwError
    * @since 2.0.0
    * @function
-   * @param {Error|String} something Thing to throw
+   * @param {Error|Object|String} something Thing to throw
    */
   SpyStrategy.prototype.throwError = function(something) {
-    var error = something instanceof Error ? something : new Error(something);
+    var error =
+      something instanceof String || typeof something === 'string'
+        ? new Error(something)
+        : something;
```

Comment:
Please add a new test for this and update/rename the test for the old behavior as appropriate.

Reply:
Done, was just waiting to see if this was a desired change before adding a test.